TASK/TaskRappelDownWall.md

```diff
@@ -1,26 +1,41 @@
 ---
 ns: TASK
 ---
-## _TASK_RAPPEL_DOWN_WALL
+## TASK_RAPPEL_DOWN_WALL
```

Comment:
This native doesn't have a Joaat hash and can therefor not be named without a `_` prefix.